### PR TITLE
Updates for changed to .NET Core 3

### PR DESCRIPTION
**Pull request by WiredUK**

### CODE REVIEW

File: StreamR.csproj [hunk 1]
```diff
@@ -7,7 +7,6 @@
 
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" Version="3.0.0-preview-19075-0444" />
     <PackageReference Include="Microsoft.AspNetCore.SignalR" Version="3.0.0-preview-19075-0444" />
```
Comment (BrennanConroy) [inline StreamR.csproj]: I think you can remove both of these references too

**Review by BrennanConroy — APPROVED** [review]
Thanks for the PR!

I have one small nit

File: StreamR.csproj (hunk 1 above)
Comment (WiredUK) [reply StreamR.csproj]: Good spot, I've removed them too.

**Comment by BrennanConroy** [comment]
Thanks again!

**Comment by WiredUK** [comment]
My pleasure, I'll be using this as a demo during my local event for .NET Conf in a couple of weeks, should be a great evening!

**Pull request merged**